<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/login.png);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>700</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background.jpg);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>208</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/hover-button.9.png);</string>
   </property>
   <property name="text">
    <string>Inciar Sesion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpPushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>208</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button.9.png);</string>
   </property>
   <property name="text">
    <string>Registrarse</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>490</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>graphicsView_2</zorder>
  <zorder>graphicsView</zorder>
  <zorder>usernameLineEdit</zorder>
  <zorder>passwordLineEdit</zorder>
  <zorder>loginPushButton</zorder>
  <zorder>signUpPushButton</zorder>
  <zorder>loginLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
